machine-learning/preprocessing.py

```diff
@@ -1,40 +1,120 @@
 # Preprocessing of text. Please visit https://spacy.io/api for more information
 import spacy
 import pandas as pd
+from spacy.symbols import nsubj, VERB, ADJ
 
+# spacy.prefer_gpu() # not very helpful for md size... slower than cpu on GTX1060
 spacy_nlp = spacy.load("en_core_web_md") # sm, md, lg
 #Trade off : md, lg give more accurate result but sm is significantly faster
 
 # Constants for word similarity comparison
-negative_word = spacy_nlp("bad")
+negative_word = spacy_nlp("disgusting")
```

Comment:
Could encapsulate these in a settings/config class. This depends on future work though (e.g. incorporating user choices), so could be done later.